packages/react-lowcode/src/codegen/generation/context/page-context.ts

```diff
@@ -31,6 +32,6 @@ export class SourceFileContext{
     }
    
     getSourceCode(): string {
-        return fs.readFileSync(this.path, 'utf-8')
+        return fs.readFileSync(path.resolve(this.path), 'utf-8')
```

Comment:
This will be probably problem while running in browser (it will work as expected from CLI)